ledger/sandbox-on-x/reference.conf

```diff
@@ -0,0 +1,563 @@
+  ledger-id = default-ledger-id
+
+  metrics {
+    # Enables metrics to be reported via defined `reporter`.
+    enabled = false
+
+    # Defines a reporter of metrics of a different type with its parameters.
+    # Options:
+    # console
+    #   Prints captured metrics to standard output.
+    # csv://</path/to/metrics.csv>
+    #   Saves the captured metrics in CSV format at the specified location.
+    # graphite://HOST[:PORT][/METRIC_PREFIX]
+    #   Sends captured metrics to a Graphite server. If the port is omitted, the default
+    #   value ``2003`` will be used. A ``metric_prefix`` can be specified, causing
+    #   all metrics to be reported with the specified prefix.
+    # prometheus://HOST[:PORT]
+    #   renders captured metrics on an HTTP endpoint in accordance with the prometheus protocol. If the port
+    #   is omitted, the default value ``55001`` will be used. The metrics will be
+    #   available under the address ``http://<server_host>:<server_port>/metrics``.
+    reporter = console
+
+    # Metrics are pre-aggregated and sent to the reporter, this option allows the user
+    # to set the interval.
+    reporting-interval = 10 seconds
+
+    # Defines the type of registry being created for each Participant configuration.
+    # Options: jvm-shared, new
+    registry-type = jvm-shared
+  }
+
+  engine {
+    # The Engine configurations describes the versions of language and
+    # transaction the engine is allowed to read and write together with
+    # engine debugging feature.
+
+    # The range of language versions the engine is allowed to load.
+    # The engine will crash if it asked to load a language version
+    # that is not included in this range.
+    # Options: daml-lf-dev-mode-unsafe, early-access, stable, legacy
+    allowed-language-versions = early-access
+
+    # Defines the behavior of the Daml engine when it encounters submissions with duplicate contract keys.
+    # Options:
+    # strict
+    #   The Daml engine rejects submissions with duplicate contract keys. This is the default option.
+    # off
+    #   Key uniqueness checks are disabled.
+    contract-key-uniqueness = strict
+
+    # TODO ???
+    forbid-v-0-contract-id = true
+
+    # This section configures limits to be enforced by the Daml engine
+    # when executing a Daml command.
+    limits {
+      # The maximum number of choice controllers allowed for Daml contracts.
+      choice-controllers = 2147483647
+
+      # The maximum number of choice observers allowed for Daml contracts.
+      choice-observers = 2147483647
+
+      # The maximum number of contract observers allowed for Daml contracts.
+      contract-observers = 2147483647
+
+      # The maximum number of contract signatories allowed for Daml contracts.
+      contract-signatories = 2147483647
+
+      # The maximum number of input contracts allowed for a Daml transaction.
+      transaction-input-contracts = 2147483647
+    }
+
+    # TODO ???
```

Comment:
@remyhaemmerle-da

Reply:
Enable package validation for loaded packages.